canal/canal_test.go

```diff
@@ -157,12 +157,12 @@ func TestRenameTableExp(t *testing.T) {
 		"rename table mydb.mytable to mydb.mytable1",
 		"rename table mytable to mytable1",
 
-		"rename table `mydb`.`mytable1` to `mydb`.`mytable2`, `mydb`.`mytable` to `mydb`.`mytable1`",
-		"rename table `mytable1` to `mytable2`, `mytable` to `mytable1`",
-		"rename table mydb.mytable1 to mydb.mytable2, mydb.mytable to mydb.mytable1",
-		"rename table mytable1 to mytable2, mytable to mytable1",
+		"rename table `mydb`.`mytable` to `mydb`.`mytable2`, `mydb`.`mytable3` to `mydb`.`mytable1`",
```

Comment:
why changing these tests?

Reply:
rename should clear old table cache,  	expRenameTable = regexp.MustCompile("(?i)^RENAME\\sTABLE\\s.*?`{0,1}(.*?)`{0,1}\\.{0,1}`{0,1}([^`\\.]+?)`{0,1}\\s{1,}TO\\s.*?")
this is change for get the old table

Reply:
got it